src/tests/system/tests/test_passkey.py

```diff
@@ -740,3 +740,151 @@ def test_passkey__su_with_12_mappings(
         )
         in output
     ), "Get the console message about TGT"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.builtwith(client="passkey", ipa="passkey")
+@pytest.mark.ticket(gh=6931)
+def test_passkey__su_no_pin_set(
+    client: Client, ipa: IPA, moduledatadir: str, testdatadir: str, umockdev_ipaotpd_update
+):
+    """
+    :title: Check authentication of user with IPA server when no pin set for the Passkey
+    :setup:
+        1. Add a user with --user-auth-type=passkey in the IPA server
+        2. Modify Passkey configuration to set require user verification during authentication to false
+        3. Setup SSSD client with FIDO and umockdev, start SSSD service
+    :steps:
+         1. Check authentication of the user when no pin set for the Passkey
+         2. Check the TGT of user
+    :expectedresults:
+        1. User authenticates successfully
+        2. Get TGT after authentication of user
+    :customerscenario: False
+    """
+    with open(f"{testdatadir}/passkey-mapping.ipa") as f:
+        ipa.user("user1").add(user_auth_type="passkey").passkey_add(f.read().strip())
+
+    ipa.host.conn.run("ipa passkeyconfig-mod --require-user-verification=False", raise_on_error=False)
+    client.sssd.start(service_user="root")
+
+    rc, _, output, _ = client.auth.su.passkey_with_output(
+        username="user1",
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.ipa",
+        command="klist",
+        auth_method="PasskeyAuthenticationUseCases.PASSKEY_NO_PIN_NO_PROMPTS",
+    )
+
+    ipa.host.conn.run("ipa passkeyconfig-mod --require-user-verification=True", raise_on_error=False)
```

Comment:
What is the reason to set back to `True` here?

Reply:
Default is `True`, I updated to `False`, reverting to `True` again.
```
[root@master ~]# ipa passkeyconfig-show 
  Require user verification: True
```

Reply:
I believe this should be handled by the Multihost role clean test state. Did you try with this line removed?

Per https://tests.sssd.io/en/latest/guides/using-roles.html

```
Role objects are created at the start of each test and destroyed when the test is finished. They create a backup of the current state of the remote host and restore modified state back to the original when the test ends.

Therefore as long as you use only the role object, you can be assure that everything you change through the role’s API is restored to its original state automatically. For example if you add a new user, it is deleted. If you create a new file, it is deleted. If you modify existing file, its content is restored.
```

Reply:
Let me try again.

Reply:
Yes, it worked, thanks!